src/conditional-flow/multiple-conditions.js

```diff
@@ -2,18 +2,18 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
-
-  // TODO: write code in this function body to pass the tests
-
+  if (num >= lower && num <= upper) {
+    return true;
+  } else { return false }
 }
 
 // This function should return true if the passed string is equal
 // to "Hello" or "Goodbye". Implement this with a single
 // if statement.
 function isHelloOrGoodbye (val1) {
-
-  // TODO: write code in this function body to pass the tests
-
+  if (val1 === 'Hello' || val1 === 'Goodbye') {
```

Comment:
Same as above, the if statement isn't needed here. You can just return `val1 === 'Hello' || val1 === 'Goodbye'`